Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -1,9 +1,455 @@
+}
+
+-- When the civ building dies
+M1SecondaryFailed = {
+    {text = '[Benson]: The building is destoryed. No intel for us.', vid = 'N03_M1SecondaryFailed.sfd', bank = 'N03_VO', cue = 'M1SecondaryFailed', faction = 'UEF'},
+}
+
+-- Remind to capture the building
+M1SecondaryReminder = {
+    {text = '[Benson]: Capture the Aeon building to gather extra intel.', vid = 'N03_M1SecondaryReminder.sfd', bank = 'N03_VO', cue = 'M1SecondaryReminder', faction = 'UEF'},
+}
+
+
+
+-- Unlock T2 static shield
+M1ShieldUnlock = {
+    {text = '[Benson]: T2 Shield uploaded.', vid = 'N03_M1ShieldUnlock.sfd', bank = 'N03_VO', cue = 'M1ShieldUnlock', faction = 'UEF'},
+}
+
+-- Bonus ojd done, construct shield over the ship
+M1ShieldConstructed = {
+    {text = '[Benson]: The shield should make sure that the ship survives longer.', vid = 'N03_M1ShieldConstructed.sfd', bank = 'N03_VO', cue = 'M1ShieldConstructed', faction = 'UEF'},
+}
+
+
+
+-- Tell the player we're sending engineers
+M1Enginners1 = {
+    {text = '[Benson]: Commander, we\'re dispatching a group of engineers to your location to investigate the ship\'s damage. They\'ll arive shortly.', vid = 'N03_M1Enginners1.sfd', bank = 'N03_VO', cue = 'M1Enginners1', faction = 'UEF'},
+}
+
+-- When the transports appear at the map
+M1Enginners2 = {
+    {text = '[Benson]: The transports are inbound.', vid = 'N03_M1Enginners2.sfd', bank = 'N03_VO', cue = 'M1Enginners2', faction = 'UEF'},
+}
+
+-- Engineers start repairing the ship
+M1Enginners3 = {
+    {text = '[Benson]: Scanning the ship for damage.', vid = 'N03_M1Enginners3.sfd', bank = 'N03_VO', cue = 'M1Enginners3', faction = 'UEF'},
+}
+
+-- Few seconds after, assign objective to reclaim crystal
+M1CrystalsObjective = {
+    {text = '[Benson]: Ship is very damaged, to repair it we must reclaim some crystals, etc etc...', vid = 'N03_M1CrystalsObjective.sfd', bank = 'N03_VO', cue = 'M1CrystalsObjective', faction = 'UEF'},
+}
+
+-- Remind player to reclaim the crystals
+M1CrystalsReminder = {
+    {text = '[Benson]: Commander, we really need those crystals, reclaim them ASAP.', vid = 'N03_M1CrystalsReminder.sfd', bank = 'N03_VO', cue = 'M1CrystalsReminder', faction = 'UEF'},
+}
+
+-- When the first crystal is reclaimed
+M1FirstCrystalReclaimed = {
+    {text = '[Nichols]: First reclaim reclaimed, get some more.', vid = 'N03_M1FirstCrystalReclaimed.sfd', bank = 'N03_VO', cue = 'M1FirstCrystalReclaimed', faction = 'UEF'},
+}
+
+-- Both crystals reclaimed, objective completed (map didnt expand yet)
+M1CrystalsReclaimed1 = {
+    {text = '[Nichols]: Both crystals reclaimed, we will need more.', vid = 'N03_M1CrystalsReclaimed1.sfd', bank = 'N03_VO', cue = 'M1CrystalsReclaimed1', faction = 'UEF'},
+}
+
+-- Both crystals reclaimed, objective completed (map has already expanded)
+M1CrystalsReclaimed2 = {
+    {text = '[Nichols]: Both crystals reclaimed.', vid = 'N03_M1CrystalsReclaimed2.sfd', bank = 'N03_VO', cue = 'M1CrystalsReclaimed2', faction = 'UEF'},
```

Comment:
Both Crystals have been reclaimed. We're incorporating the materials into the ship structure now.

Needs to be structured correctly.